internal/browser/browser.go

```diff
@@ -41,10 +41,19 @@ func getWindowsChromeLocation() string {
 		return "C:/Program Files (x86)/Google/Chrome/Application/chrome.exe"
```

Comment:
I'm pretty unhappy with how I added all this code in to begin with, we probably want to refactor the function to return a string or an err and then do 

if check loc 1
  return loc1, nil
if check loc 2
 return loc2, nil
if check loc 3
 return loc3, nil

log.fatal(foobar)
return nil, err

Reply:
Sure, will update it tomorrow morning.